clients/include/testing_gbmv_batched.hpp

```diff
@@ -66,143 +62,154 @@ hipblasStatus_t testing_gbmvBatched(const Arguments& argus)
         return HIPBLAS_STATUS_SUCCESS;
     }
 
-    hipblasHandle_t handle;
-    hipblasCreate(&handle);
+    hipblasLocalHandle handle(argus);
 
-    double gpu_time_used, cpu_time_used;
-    double hipblasGflops, cblas_gflops, hipblasBandwidth;
-    double rocblas_error;
+    double gpu_time_used, hipblas_error_host, hipblas_error_device;
 
-    T alpha = (T)argus.alpha;
-    T beta  = (T)argus.beta;
+    T h_alpha = (T)argus.alpha;
```

Comment:
Nan testing something for the future, but could use accessor even before we try it.  A different PR.